bug: test_integration_duplicate_detection_with_file_processing が CI で断続的に失敗する

## 概要

`tests/unit/workers/test_database_worker.py::TestRegistrationErrorHandling::test_integration_duplicate_detection_with_file_processing` が CI 環境で断続的に失敗する。ローカルでは修正前・修正後ともに PASS。

## エラー内容

```
FAILED tests/unit/workers/test_database_worker.py::TestRegistrationErrorHandling::test_integration_duplicate_detection_with_file_processing - assert 0 == 1
```

## 確認済み事項

- ローカル（Python 3.13, Linux WSL2）では常に PASS
- CI 環境（Python 3.12.13, ubuntu-latest）で FAIL（PR #127 の CI ログで確認）
- Issue #118 の修正（`_apply_nsfw_filter` タグ追加）とは無関係

## 推定原因

- CI 環境の Python バージョン差異（3.12 vs 3.13）
- Qt ヘッドレス環境でのタイミング問題
- ファイルシステム操作の競合（一時ディレクトリ）

## 関連ファイル

- `tests/unit/workers/test_database_worker.py` (`TestRegistrationErrorHandling` クラス)

## 確認方法

```bash
uv run pytest tests/unit/workers/test_database_worker.py::TestRegistrationErrorHandling::test_integration_duplicate_detection_with_file_processing -v
```

--- a/src/lorairo/gui/workers/registration_worker.py
+++ b/src/lorairo/gui/workers/registration_worker.py
@@ -5,6 +5,8 @@ from dataclasses import dataclass
 from pathlib import Path
 from typing import TYPE_CHECKING
 
+from sqlalchemy.exc import SQLAlchemyError
+
 from ...annotations.existing_file_reader import ExistingFileReader
 from ...utils.log import logger
 from .base import LoRAIroWorkerBase
@@ -184,7 +186,11 @@ class DatabaseRegistrationWorker(LoRAIroWorkerBase[DatabaseRegistrationResult]):
                 image_path, duplicate_image_id, annotations=annotations, tag_id_cache=tag_id_cache
             )
             # ファイル名エイリアスを登録（バッチインポート時のマッチング用）
-            self.db_manager.repository.add_filename_alias(duplicate_image_id, image_path.stem)
+            # DBが未初期化の場合でも重複扱い（skipped）は変わらないため例外を握り潰す
+            try:
+                self.db_manager.repository.add_filename_alias(duplicate_image_id, image_path.stem)
+            except SQLAlchemyError as e:
+                logger.warning(f"エイリアス登録失敗（スキップ扱いは継続）: {image_path.stem}, {e}")
             logger.debug(f"スキップ (重複): {image_path} - 関連ファイルは処理")
             result_type = "skipped"
             image_id = duplicate_image_id

--- a/tests/unit/workers/test_database_worker.py
+++ b/tests/unit/workers/test_database_worker.py
@@ -1026,6 +1026,34 @@ class TestRegistrationErrorHandling:
             # スキップ数が増加することを確認
             assert result.skipped_count == 1
 
+    def test_duplicate_skipped_even_if_alias_registration_fails(self, temp_dir, real_db_manager, mock_fsm):
+        """add_filename_alias が SQLAlchemyError を送出しても skipped_count が増加する"""
+        from sqlalchemy.exc import SQLAlchemyError
+
+        image_file = temp_dir / "alias_fail_test.jpg"
+        tag_file = temp_dir / "alias_fail_test.txt"
+        image_file.write_bytes(b"fake_image")
+        tag_file.write_text("tag1", encoding="utf-8")
+        mock_fsm.get_image_files.return_value = [image_file]
+
+        with (
+            patch.object(real_db_manager, "detect_duplicate_image") as mock_detect,
+            patch.object(real_db_manager, "save_tags"),
+            patch.object(real_db_manager, "save_captions"),
+            patch.object(
+                real_db_manager.repository,
+                "add_filename_alias",
+                side_effect=SQLAlchemyError("table not found"),
+            ),
+        ):
+            mock_detect.return_value = 999
+
+            worker = DatabaseRegistrationWorker(temp_dir, real_db_manager, mock_fsm)
+            result = worker.execute()
+
+            assert result.skipped_count == 1
+            assert result.error_count == 0
+
     def test_high_volume_image_processing(self, temp_dir, real_db_manager):
         """100個の画像処理時のパフォーマンステスト"""
         # 100個の画像ファイルを作成

